xarray/core/computation.py

```diff
@@ -678,7 +678,7 @@ def apply_variable_ufunc(
                                     "apply_ufunc with dask='parallelized' consists of "
                                     "multiple chunks, but is also a core dimension. To "
                                     "fix, either rechunk into a single dask array chunk along "
-                                    f"this dimension, i.e., ``.chunk({dim}: -1)``, or "
+                                    f"this dimension, i.e., ``.chunk(dict({dim}=-1))``, or "
```

Comment:
```suggestion
                                    f"this dimension, i.e., ``.chunk(dict(dim=-1))``, or "
```

Reply:
But then it's not copy/paste friendly? 
I got this error a few times in the cross product pr and I felt it was a nice and helpful idea to just be able to append `.chunk(dict(cartesian=-1))` without having to think.

Reply:
Sorry! I misread this. Your idea is great

Reply:
note that this is not copy-paste friendly in general: `dict(1=-1)` is invalid python